website/docs/r/guardduty_detector.html.markdown

```diff
@@ -26,7 +25,7 @@ resource "aws_guardduty_detector" "MyDetector" {
 The following arguments are supported:
 
 * `enable` - (Optional) Enable monitoring and feedback reporting. Setting to `false` is equivalent to "suspending" GuardDuty. Defaults to `true`.
-* `finding_publishing_frequency` - (Optional) Specifies the frequency of notifications sent for subsequent finding occurrences. Valid values: `FIFTEEN_MINUTES, ONE_HOUR, SIX_HOURS`. Default: `SIX_HOURS`. See [AWS Documentation](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_findings_cloudwatch.html#guardduty_findings_cloudwatch_notification_frequency) for more information.
+* `finding_publishing_frequency` - (Optional) Specifies the frequency of notifications sent for subsequent finding occurrences. Cannot be modified for GuardDuty member accounts. Must be configured to enable drift detection in GuardDuty master accounts. Valid values: `FIFTEEN_MINUTES`, `ONE_HOUR`, `SIX_HOURS`. Default: `SIX_HOURS` for GuardDuty master account or matches GuardDuty master account if GuardDuty member account. See [AWS Documentation](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_findings_cloudwatch.html#guardduty_findings_cloudwatch_notification_frequency) for more information.
```

Comment:
> Valid values: `FIFTEEN_MINUTES`, `ONE_HOUR`, `SIX_HOURS`. Default: `SIX_HOURS` for GuardDuty master account or matches GuardDuty master account if GuardDuty member account.

I'm not sure what you are tying to say here. Is this supposed to say something along the lines of `GuardDuty member account will match the frequency value defined in the master account`?

Reply:
Yeah, I'll try to clarify the wording a little better. 👍